### ISSUE TITLE
Details of training base feature extractor (src/utils/train_image.py)

### ISSUE THREAD

**Issue by Qingzhang1994** [AnjanDutta/sem-pcyc#33]
When I trained the code, I found that the function required by the following code were missing. Could you please provide the code to help me study?
weights = torch.DoubleTensor(utils.class_weights(tr_clss)) in line 116
acc = utils.accuracy(op.data, cl.data, topk=(1,)) in line 276.

I found some functions to calculate accuracy, but after training an epoch it appears that the training set has 99% accuracy and the verification set has 0% accuracy.
![image](https://user-images.githubusercontent.com/48178899/224586985-2a6bc7ea-1352-4c3f-b6d6-66ae0032602d.png)
